src/modules/auth/auth.routes.js

```diff
@@ -1,8 +1,8 @@
 import express from 'express';
-import AuthController from './auth.controller';
-import ValidateAuth from '../../middlewares/ValidateAuth';
+import * as AuthController from './auth.controller';
+import * as AuthMiddleware from '../../middlewares/auth.middleware';
```

Comment:
Try not to use PascalCase for functions

Reply:
Will do, but am exporting multiple functions, is that still ok ?